content/telegraf/v1.24/configure_plugins/custom_builder.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Reduce Telegraf binary size with customized agent
+description: Use the custom builder tool to reduce Telegraf binary disk space. 
+menu:
+  telegraf_1_24:
+
+    name: Reduce binary size
+    Parent: Configure plugins
+    weight: 20
+---
+Reduce Telegraf's overall memory and disk footprint with the custom builder. This tool allows you to select which plugins to use in the Telegraf binary based on your configuration file. 
+
+## Requirements
+
+- Ensure you've installed the current version of [Go](https://go.dev/). 
+- Create your Telegraf configuration file with the plugins you want to use. For details, see [Configuration options](/telegraf/v1.24/configuration/).
+
+## Build and run the custom builder
+
+1. Clone the Telegraf repository:
+    ```
+    git clone https://github.com/influxdata/telegraf.git
+    ```
+2. Change directories into the top-level of the Telegraf repository:
+    ```
+    cd telegraf
+    ```
+3. Build the Telegraf custom builder tool by entering the folllowing command:
+    ```
+    make build_tools
+    ```
```

Comment:
~~~suggestion
3. Build the Telegraf custom builder tool by entering the folllowing command:
    ```
    make build_tools
    ```

    This process stores the `custom_builder` utility at `./tools/custom_builder/custom_builder`
~~~